dashboard/lib/pd/foorm/survey_reporter.rb

```diff
@@ -157,12 +161,18 @@ def self.get_raw_data_for_workshop(workshop_id, facilitator_id=nil)
     end
 
     # @param integer workshop_id: id for a workshop
+    # @param integer facilitator_filter: If specified, only get facilitator data for the facilitator
+    #   with this id
     # @return {facilitator_id: facilitator_name,...} object with data
     # for each facilitator for the workshop specified
-    def self.get_formatted_facilitators_for_workshop(workshop_id)
+    def self.get_formatted_facilitators_for_workshop(workshop_id, facilitator_filter=nil)
```

Comment:
Maybe `facilitator_filter` could be something like `facilitator_id_filter` or `filter_facilitator_id` to remind us that it contains a facilitator ID?

Reply:
good call